x-pack/test/functional/services/ml/test_resources.ts

```diff
@@ -70,6 +70,10 @@ export function MachineLearningTestResourcesProvider({ getService }: FtrProvider
       return this.getSavedObjectIdByTitle(title, SavedObjectType.SEARCH);
     },
 
+    async getDashboardId(title: string): Promise<string | undefined> {
```

Comment:
Heads-up: I've also added this one in #69251. Depending on merge order, this might cause a conflict.